From bb725b4a18969fee5c955ec5dde3229d4219512c Mon Sep 17 00:00:00 2001
From: suvojeet-sengupta <suvojitsengupta21@gmail.com>
Date: Tue, 6 Oct 2026 22:27:23 +0000
Subject: [PATCH] a2dp: Don't log every codec config byte

copyOutOtaCodecConfig logged each of the AVDT_CODEC_SIZE bytes at INFO
on every call, hundreds of lines a minute while an A2DP device is
connected.

Change-Id: I8af329d17f1698971f1c6cf486a707a988195f45
---
 system/stack/a2dp/a2dp_codec_config.cc | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/system/stack/a2dp/a2dp_codec_config.cc b/system/stack/a2dp/a2dp_codec_config.cc
index a0eaca3245..cd4d2962bc 100644
--- a/system/stack/a2dp/a2dp_codec_config.cc
+++ b/system/stack/a2dp/a2dp_codec_config.cc
@@ -348,10 +348,6 @@ bool A2dpCodecConfig::copyOutOtaCodecConfig(uint8_t* p_codec_info) {
                << (float)ota_codec_config_[i];
   }
   memcpy(p_codec_info, ota_codec_config_.data(), AVDT_CODEC_SIZE);
-  for (int i = 0; i < AVDT_CODEC_SIZE; i++) {
-    LOG(INFO) << __func__ << ": type of p_codec_info[" << i << "]: " << ", p_codec_info[" << i
-               << "]: " << (float)p_codec_info[i];
-  }
   return true;
 }
 
-- 
2.43.0

